src/initialize_city.py

```diff
@@ -73,15 +73,20 @@ def make_config_file(yml_file, city, folder, crash, concern, supplemental=[]):
 
         for filename in supplemental:
             f.write(
-                "  - name: parking_tickets\n" +
+                "  - name: \n" +
                 "    filename: {}\n".format(filename) +
                 "    address: \n" +
                 "    date: \n" +
                 "    time: \n" +
                 "    category: \n" +
                 "    notes: \n" +
                 "    # Feature is categorical (f_cat) or continuous (f_cont)\n" +
-                "    feat: \n")
+                "    feat: \n" + 
+                "    #feat_agg (feature aggregation) can be total count (default) or latest value \n" +
+                "    feat_agg: \n"
+                "    # if latest, the desired value \n" +
```

Comment:
What do you mean by desired value?

Reply:
This is the place to store the actual value in the point json blob. So if the traffic volume for x street in Philly is 2000 units, then 2000 would go there. Is this overlapping with another column?

Reply:
Before with counts or category counts, there wasn't a need to store actual values (from what I can gather! this could def be wrong)

Reply:
I think it would be clearer if your comment reads column name where the value is found, or something like that. Because RECORDNUM isn't the value, it's how you look up the value. I might also call the variable something else, but I don't feel strongly about that as long as the comment makes it clear.